Samples/Sample.Xamarin.Core/Helpers/SentryInitializer.cs

```diff
@@ -1,21 +1,18 @@
 ﻿using Sentry;
-using Sentry.Xamarin.Forms;
-using System;
-using System.IO;
 
 namespace Sample.Xamarin.Core.Helpers
 {
     public static class SentryInitializer
     {
-        public static void Init(bool enableCache = true)
+        public static void Init()
         {
-            var path = enableCache ? Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData) : null;
-            SentrySdk.Init(o =>
+            SentryXamarin.Init(options =>
             {
-                o.CacheDirectoryPath = path;
-                o.Debug = true;
-                o.Dsn = "https://5a193123a9b841bc8d8e42531e7242a1@o447951.ingest.sentry.io/5560112";
-                o.AddIntegration(new SentryXamarinFormsIntegration());
+                options.Dsn = "https://5a193123a9b841bc8d8e42531e7242a1@o447951.ingest.sentry.io/5560112";
+#if DEBUG
+                options.DiagnosticLogger = new Sentry.Infrastructure.DebugDiagnosticLogger(SentryLevel.Debug);
```

Comment:
We could try ti make the main sdk smarter and add the debug logger instead of console automatically